scripts/deploy-sdks.ts

```diff
@@ -0,0 +1,285 @@
+#!/usr/local/bin/node
+
+/*
+ * deploy-sdks script
+ * THIS SCRIPT MUST BE RUN WITH NPM TO PUBLISH - `npm run deploy-sdks`
+ * From the monorepo root run `yarn deploy-sdks`
```

Comment:
This should be `npm run deploy-sdks` instead. I know you copied this but I changed it in the monorepo too.